internal/server/iigointernal/judiciary_test.go

```diff
@@ -263,7 +267,7 @@ func TestInspectHistory(t *testing.T) {
 			}, getBaseEvalResults(shared.TeamIDs)),
 			expectedSuccess: true,
 		},
-		{
+		/*{
```

Comment:
Hmmmm sus

Reply:
Sorry, didn't mean to commit that. It was added alongside the other tests I fixed.